### ISSUE TITLE
Example of resolveTransaction on hw-app-eth/README.md incorrect

### ISSUE THREAD

**Issue by playmint-jbr** [LedgerHQ/ledgerjs#788]
The example of how to use resolveTransaction on https://github.com/LedgerHQ/ledgerjs/blob/master/packages/hw-app-eth/README.md just passes the unsigned tx, but the real function has 3 args.

**Comment by bkbooth** [comment]
Yep, `hw-app-eth` is now showing a deprecation warning about not providing a `resolution` to `signTransaction`, but the example given in the docs doesn't work.
